Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfBufferViewTarget.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// The target that the GPU buffer should be bound to.
+    /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/bufferView.schema.json
+    /// </summary>
+    public enum GltfBufferViewTarget
+    {
+        None = 0,
```

Comment:
Can we get summary tags for these enum values? Would help customers know what they should use,

Reply:
Individual enum declarations didn't exist